CMakeLists.txt

```diff
@@ -185,6 +187,26 @@ FOREACH (currentBin ${LIST_COMPONENTS})
     endif()
 ENDFOREACH(currentBin)
 
+FOREACH (currentBin ${LIST_DRIVERS})
+    if (EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/src/stable/drivers/${currentBin}/${currentBin})
+        INSTALL (PROGRAMS ${CMAKE_CURRENT_SOURCE_DIR}/src/stable/drivers/${currentBin}/${currentBin} DESTINATION /usr/local/bin OPTIONAL COMPONENT core)
+    endif()
+    if (EXISTS ${CMAKE_CURRENT_BINARY_DIR}/src/stable/drivers/${currentBin}/${currentBin})
+        INSTALL (PROGRAMS ${CMAKE_CURRENT_BINARY_DIR}/src/stable/drivers/${currentBin}/${currentBin} DESTINATION /usr/local/bin OPTIONAL COMPONENT core)
+    endif()
+ENDFOREACH(currentBin)
+
+FOREACH (currentBin ${LIST_TOOLS})
+    if (EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/src/stable/tools/${currentBin}/${currentBin})
+        INSTALL (PROGRAMS ${CMAKE_CURRENT_SOURCE_DIR}/src/stable/toolss/${currentBin}/${currentBin} DESTINATION /usr/local/bin OPTIONAL COMPONENT core)
```

Comment:
Here's a little mistake that is preventing me from the installation of the tools and the package generation! The path says "toolss" instead of "tools" so it will not install any tool in /usr/local/bin. Hotfix incoming :)